pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -123,7 +130,7 @@ func (am *Alertmanager) Init() (err error) {
 		return errors.Wrap(err, "unable to initialize the silencing component of alerting")
 	}
 
-	am.alerts, err = NewAlertProvider(am.marker)
+	am.alerts, err = mem.NewAlerts(context.Background(), am.marker, 30*time.Minute, gokit_log.NewNopLogger())
```

Comment:
// memoryAlertsGCInterval is the interval at which we'll remove resolved alerts from memory.
const memoryAlertsGCInterval = 30*time.Minute